x-pack/plugins/apm/public/context/url_params_context/helpers.ts

```diff
@@ -19,6 +19,17 @@ function getParsedDate(rawDate?: string, options = {}) {
   }
 }
 
+export function getExactDate(rawDate?: string, options = {}) {
```

Comment:
Can we make the first arg required and remove the second?

```suggestion
export function getExactDate(rawDate: string) {
```